consensus/hotstuff/cruisectl/config.go

```diff
@@ -2,57 +2,81 @@ package cruisectl
 
 import (
 	"time"
+
+	"go.uber.org/atomic"
 )
 
-// DefaultConfig returns the default config for the BlockRateController.
+// DefaultConfig returns the default config for the BlockTimeController.
 func DefaultConfig() *Config {
 	return &Config{
-		TargetTransition: DefaultEpochTransitionTime(),
-		// TODO confirm default values
-		DefaultProposalDuration: 500 * time.Millisecond,
-		MaxProposalDuration:     1000 * time.Millisecond,
-		MinProposalDuration:     250 * time.Millisecond,
-		Enabled:                 true,
-		N_ewma:                  5,
-		N_itg:                   50,
-		KP:                      2.0,
-		KI:                      0.6,
-		KD:                      3.0,
+		TimingConfig{
+			TargetTransition:      DefaultEpochTransitionTime(),
+			FallbackProposalDelay: atomic.NewDuration(500 * time.Millisecond),
```

Comment:
```suggestion
			FallbackProposalDelay: atomic.NewDuration(250 * time.Millisecond),
```
As discussed, I think it's worthwhile to lower the fallback delay a bit. Since it is so much easier for the controller to compensate for negative error than positive error. I have tested with 250ms and a 30% controlled cohort is able to maintain equilibrium: https://www.notion.so/dapperlabs/Test-6-Uncontrolled-cohort-has-0-proposal-delay-5d5a71c116694994a305baef8b1a1b71?pvs=4